go/engine/wot_vouch.go

```diff
@@ -53,8 +53,24 @@ func (e *WotVouch) SubConsumers() []libkb.UIConsumer {
 	return nil
 }
 
+func getSigIDsToRevoke(mctx libkb.MetaContext, vouchee *libkb.User) (toRevoke []keybase1.SigID, err error) {
```

Comment:
So vouch_with_revoke revokes all previous vouches?

Reply:
pushed a commit to clarify this